src/main/java/core/basesyntax/service/impl/ReportMakerServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportMakerService;
+import java.util.Map;
+
+public class ReportMakerServiceImpl implements ReportMakerService {
+
+    private static final String HEAD_LINER = "fruit, quantity";
+
+    @Override
+    public String report(Map<String, Integer> report) {
+        StringBuilder builder = new StringBuilder();
+        builder.append(HEAD_LINER);
+        report.forEach((key, value) -> builder
+                .append(System.lineSeparator())
+                .append(key).append(", ").append(value));
+        return builder.toString();
```

Comment:
can we collect our data without the `StringBuilder` or concatenation using `Collectors.joining()`? would be more clean
[hint](https://www.geeksforgeeks.org/java-8-streams-collectors-joining-method-with-examples/)